.github/workflows/codeql.yml

```diff
@@ -53,6 +54,11 @@ jobs:
       #   echo "Run, Build Application using script"
       #   ./location_of_script_within_repo/buildscript.sh
 
+      - name: Build .NET code
+        if: ${{ (matrix.language == 'csharp') }}
+        working-directory: ${{github.workspace}}/dotnet
+        run: dotnet build /p:EnableWindowsTargeting=true
```

Comment:
Ja, for the time being, this is necessary - but if that's all that it takes,  then fine with me.